### PR TITLE
Fixes bug 1114552 - Added Query service to es module.

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
@phrawzty r?

### CODE REVIEW

File: socorro/external/es/query.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        except ValueError:
+            raise BadArgumentError(
+                'query',
+                msg="Invalid JSON value for parameter 'query'"
+            )
+
+        # Set indices.
+        indices = []
+        if not params.indices:
+            # By default, use the last two indices.
```
Comment (AdrianGaudebert) [inline socorro/external/es/query.py]: The UI actually even fills the input with the last two indices by default.

File: socorro/external/es/query.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            today = datetimeutil.utc_now()
+            last_week = today - datetime.timedelta(days=7)
+
+            indices = self.generate_list_of_indexes(last_week, today)
+        elif len(params.indices) == 1 and params.indices[0] == 'ALL':
+            # If we want all indices, just do nothing.
+            pass
+        else:
```
Comment (AdrianGaudebert) [inline socorro/external/es/query.py]: It will fail because, most probably, ``ALL`` is not a valid index.

File: socorro/external/es/query.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            results = connection.search(
+                body=query,
+                **search_args
+            )
+        except elasticsearch.exceptions.NotFoundError, e:
+            missing_index = re.findall(BAD_INDEX_REGEX, e.error)[0]
+            raise ResourceNotFound(
+                "elasticsearch index '%s' does not exist" % missing_index
+            )
+        except elasticsearch.exceptions.TransportError, e:
```
Comment (AdrianGaudebert) [inline socorro/external/es/query.py]: I'm not too keen on changing the features during this rewrite, as you know. Plus I am not sure what the value of a retry would be here, like, if we add it here why not add it to all other services? I'm sure users don't mind hitting F5 that much. :)

**Comment by AdrianGaudebert** [comment]
@phrawzty comment addressed!

**Comment by phrawzty** [comment]
`r+` pending Jenkins. :fish_cake:

**Pull request merged**